docs/development/configuration.md

```diff
@@ -20,6 +20,9 @@ Options which have `"globalOnly": true` are reserved only for bot global configu
 You can override default configuration using a configuration file, with default name `config.js` in the working directory.
 If you need an alternate location or name, set it in the environment variable `RENOVATE_CONFIG_FILE`.
 
+**Note:** `RENOVATE_CONFIG_FILE` must be provided with an explicit file extension.
```

Comment:
```suggestion
**Note:** `RENOVATE_CONFIG_FILE` must be provided with an explicit file extension.
For example `RENOVATE_CONFIG_FILE=myconfig.js` and not `RENOVATE_CONFIG_FILE=myconfig`.
```

Reply:
Thanks! Maybe even:

> For example `RENOVATE_CONFIG_FILE=myconfig.js` or `RENOVATE_CONFIG_FILE=myconfig.json`, and not `RENOVATE_CONFIG_FILE=myconfig`.

Based on @viceice's comment in https://github.com/renovatebot/renovate/pull/12644#discussion_r758102275?